pset-iterators.js

```diff
@@ -92,7 +101,11 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentialParties = presidents.map((presParties) => {
```

Comment:
@cbrooks2019 nice use of `.map()` - instead of naming your iterator `presParties`, you would be better served following the convention in class, simply using the singular version of the plural array name (`presidents`) - so naming your iterator `president` would make your code easier to read